Release 2.0.0 (#2766)
timescale/timescaledb: CHANGELOG.md

  permissive) rights to users and developers.

Some of the changes above (e.g., continuous aggregates, updated 
informational views) do introduce breaking changes to APIs and are not 
backwards compatible. While the update scripts in TimescaleDB 2.0 will 
upgrade databases running TimescaleDB 1.x automatically, some of these 
API and feature changes may require changes to clients and/or upstream 
scripts that rely on the previous APIs.  Before upgrading, we recommend
reviewing upgrade documentation at docs.timescale.com for more details.

**Major Features**

TimescaleDB 2.0 moves the following major features to GA:
* #1923 Add support for distributed hypertables
* #2006 Add support for user-defined actions
* #2125 #2221 Improve Continuous Aggregate API
* #2084 #2089 #2098 #2417 Redesign informational views
* #2435 Move enterprise features to community
* #2437 Update Timescale License

**Previous Release Candidates**

* #2702 Release Candidate 4 (December 2, 2020)
* #2637 Release Candidate 3 (November 12, 2020)
* #2554 Release Candidate 2 (October 20, 2020)
* #2478 Release Candidate 1 (October 1, 2020)

**Minor Features**

> Since the last release candidate 4, it includes several minor 

Reviewer: I suggest the following reformulation since "it" refers to the current release, which is not mentioned in the sentence:

```suggestion
Since the last release candidate 4, there are several minor 
```

improvements:
* #2746 Optimize locking for create chunk API
* #2705 Block tableoid access on distributed hypertable
* #2730 Do not allow unique index on compressed hypertables
* #2764 Bootstrap data nodes with versioned extension

**Bugfixes**

> Since the last release candidate 4, it includes several bugfixes:

Reviewer: ```suggestion
Since the last release candidate 4, there are several bugfixes:
```
